puppet/zulip_ops/manifests/postgres_common.pp

```diff
@@ -25,7 +26,7 @@
     minute => 0,
     target => "postgres",
     user => "postgres",
-    require => [ File["/usr/local/bin/pg_backup_and_purge.py"], Package["postgresql-${zulip::base::postgres_version}", "python-dateutil"] ]
+    require => [ File["/usr/local/bin/pg_backup_and_purge.py"], Package["postgresql-${zulip::base::postgres_version}", "python3-dateutil", "python-dateutil"] ]
```

Comment:
(Ha, aka earlier in this same file. I had that other instance already but not this one. Anyway, fixed now.)